### PR TITLE
SEPARATED Security Key to .env file and resolved the "debug always turned on in production" problem using decouple and config 

### PR DESCRIPTION
**Pull request by kingrishabdugar**

### CODE REVIEW

File: downloader/settings.py [hunk 1]
```diff
@@ -11,23 +11,24 @@
 """
 
 import os
+from decouple import config
+from decouple import config, Csv
```
Comment (Phantsure) [inline downloader/settings.py]: why have you duplicated this line?

File: downloader/wsgi.py [hunk 2]
```diff
@@ -12,5 +12,4 @@
 from django.core.wsgi import get_wsgi_application
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "downloader.settings")
-
```
Comment (Phantsure) [inline downloader/wsgi.py]: no need to remove this line

**Review by Phantsure — CHANGES_REQUESTED** [review]
add ".env.example" to show how to form ".env"

File: downloader/settings.py (hunk 1 above)
Comment (kingrishabdugar) [reply downloader/settings.py]: Actually at first I was trying with Allowed hosts .Sorry I m correcting it

**Review by Phantsure — APPROVED** [review]
works fine. would be good you could add .env to .gitignore

**Comment by Phantsure** [comment]
solves #9

**Pull request merged**